Formula/pick.rb

```diff
@@ -0,0 +1,20 @@
+class Pick < Formula
+  desc "Utility to choose one option from a set of choices"
+  homepage "https://github.com/thoughtbot/pick"
+  url "https://github.com/thoughtbot/pick/releases/download/v1.4.0/pick-1.4.0.tar.gz"
+  sha256 "46f46b0df54cf27e8dd19ae291d5534cb55ef37d9cdb3cc774cd88c809f718fd"
+
+  depends_on "autoconf" => :build
```

Comment:
I think so too. For reference here's the formula we're currently using in the thoughtbot tap: https://github.com/thoughtbot/homebrew-formulae/blob/3e52b878635553c9a493808782151e41a3fdb99b/Formula/pick.rb